src/execution/helper.rs

```diff
@@ -34,6 +35,10 @@ pub struct ExecutionHelper {
     // Pointer to the Cairo ExecutionInfo struct of the current call.
     // Must match the ExecutionInfo pointer for system call validation in 'enter_call'
     pub call_execution_info_ptr: Option<Relocatable>,
+    // The block number and block hash of the (current_block_number - buffer) block, where
+    // buffer=STORED_BLOCK_HASH_BUFFER.
+    // It is the hash that is going to be written by this OS run.
+    pub old_block_number_and_hash: Option<(Felt252, Felt252)>,
```

Comment:
When is it supposed to be `None`?

Reply:
That's a good question, it's just optional in the Python VM so I kept the implementation. Seems weird to me as well.